bricks/basic-bricks/src/app-bar-breadcrumb/index.tsx

```diff
@@ -36,12 +36,10 @@ export class AppbarBreadcrumbElement extends UpdatingElement {
   breadcrumb: BreadcrumbItemConf[];
 
   /**
-   * @default false
+   * @default -
    * @description 是否隐藏当前应用名称
    */
-  @property({
-    type: Boolean,
-  })
+  @property()
```

Comment:
不填入的话，默认为 `type: String`

```suggestion
  @property({ attribute: false })
```